dartagnan/src/main/java/com/dat3m/dartagnan/encoding/SymmetryEncoder.java

```diff
@@ -39,8 +41,9 @@ public class SymmetryEncoder implements Encoder {
     private final Relation rel;
 
     @Option(name = BREAK_SYMMETRY_ON_RELATION,
-            description = "The relation on which symmetry breaking should happen." +
-            "Empty, if symmetry shall not be encoded.",
+            description = "The relation which should be used to break symmetry." +
+            "Empty, if symmetry shall not be encoded." +
+            "Special value \"_cf\" allows for breaking on control flow rather than relations.",
             secure = true)
```

Comment:
While implementation-wise, this is always a relation, at the abstract level it is not. I would rename the option name to `encoding.symmetry.breakOn`, and use `Concept (any better option) to break symmetry on. Allowed options ... (descriptions). Empry if symmetry shall not be used`.

What happen if someone uses a relation different than `rf` and `co`?

Reply:
Regarding your question, you can use any relation you want. If the specified relation is not existent in the memory model, a warning is issued and the symmetry breaking gets disabled. In particular, for all non-sensical inputs, the symmetry breaking gets disabled.